xcodeproj.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'claide',         '>= 1.0.2', '< 2.0'
   s.add_runtime_dependency 'colored2',       '~> 3.1'
   s.add_runtime_dependency 'nanaimo',        '~> 0.3.0'
-  s.add_runtime_dependency 'rexml',          '~> 3.2.4'
+  s.add_runtime_dependency 'rexml',          '~> 3.3.2'
```

```suggestion
  s.add_runtime_dependency 'rexml',          '>= 3.2.4', '< 4.0'
```

Would this be helpful to allow users to upgrade to newer minor and patch releases in the future?

---

Done. Can this get done quick? Taking a long time for a trivial but big security concern.

---

I am waiting on this merging too, I do not have maintainer rights to merge this.